articles/getting-started-with-linux-container-security/index.md

```diff
@@ -0,0 +1,62 @@
+# Getting Started With Linux Containers Security
+
+### Overview of containers
+Linux containers, commonly referred to as LXC, are operating systems used to run multiple containers together using a single Linux kernel through a control host. In containers, users can create and manage application containers and systems easily as the API that is sturdy and uses tools that are easy to use.
```

Comment:
Could you break this sentence `In containers, users can create and manage application containers and systems easily as the API that is sturdy and uses tools that are easy to use.` into two. In seems difficult to understand.